packages/editor/src/components/block-list/block-contextual-toolbar.js

```diff
@@ -13,7 +13,7 @@ function BlockContextualToolbar() {
 	return (
 		<NavigableToolbar
 			className="editor-block-contextual-toolbar"
-			aria-label={ __( 'Block Toolbar' ) }
+			aria-label={ __( 'Block tools' ) }
```

Comment:
I'd agree for the need of guidelines. However this is invisible text and only announced by screen readers. Uppercase letters tend to be announced by screen readers increasing the voice "pitch" especially when reading letter by letter. We'd want to avoid that. I'd recommend to use normal sentence case for any invisible text meant for screen readers usage.

Now that I think at that, this and the aria-labels in #11155 need translators comments, I'm going to add them. See similar comments (though they should be improved a bit):

https://github.com/WordPress/gutenberg/blob/60e0b50a33f81cf0be90c8953862f39d7bdbbf3a/packages/edit-post/src/components/layout/index.js#L62

https://github.com/WordPress/gutenberg/blob/60e0b50a33f81cf0be90c8953862f39d7bdbbf3a/packages/edit-post/src/components/layout/index.js#L76

https://github.com/WordPress/gutenberg/blob/60e0b50a33f81cf0be90c8953862f39d7bdbbf3a/packages/edit-post/src/components/header/index.js#L36

https://github.com/WordPress/gutenberg/blob/60e0b50a33f81cf0be90c8953862f39d7bdbbf3a/packages/edit-post/src/components/sidebar/settings-header/index.js#L21-L30

https://github.com/WordPress/gutenberg/blob/60e0b50a33f81cf0be90c8953862f39d7bdbbf3a/packages/format-library/src/link/inline.js#L47